api/tests/test_playthroughs.py

```
@@ -0,0 +1,2836 @@
+        "notes": "Amazing game, loved every minute!",
+    }
+
+    response = client.post(
+        "/api/v1/playthroughs", json=playthrough_data, headers={"X-User-Id": "user-1"}
+    )
+    assert response.status_code == 201
+
+    data = response.json()
+    assert data["status"] == "COMPLETED"
+    assert data["rating"] == 9
+    assert data["completed_at"] == "2024-02-15T18:30:00Z"
```

Ruff (0.12.2)

Use of `assert` detected

(S101)